contracts/Prover.sol

```diff
@@ -106,6 +109,13 @@ contract Prover is SimpleProver {
         uint256 indexed _destinationChainID, uint256 indexed _blockNumber, bytes32 _L2WorldStateRoot
     );
 
+    /**
+     * @notice emitted on a proving state if the blockNumber is less than or equal to the current blockNumber + SETTLEMENT_BLOCKS_DELAY
+     * @param _inputBlockNumber the block number we are trying to prove
+     * @param _neededBlockNumber the  next block number that can been proven
```

Comment:
SUPER ULTRA MEGA NIT: extra space between 'the' and 'next' on line 115

Reply:
more nits: 'that can been proven' --> 'that can be proven'
also i dont love the name of this param, can we do like _earliestProvableBlockNumber maybe? doesnt actually matter though

Reply:
Nice catch, updated now :pray: